File: backend/src/main/java/com/demo/student_management/dto/auth/ProfileResponse.java
package com.demo.student_management.dto.auth;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProfileResponse {
    private String idTaiKhoan;
    private String tenDangNhap;
    private String loaiTaiKhoan;
    private String ten;
    private String cmnd;
    private String sdt;
    private String ngaySinh;
    private String gioiTinh;
    private String chuNhiemLopId;
    private List<String> phanCongKeys;
}

File: frontend/StudentManagement.Shared/Dtos/Auth/ProfileResponse.cs
using System;
using System.Collections.Generic;

namespace StudentManagement.Shared.Dtos.Auth;

public sealed class ProfileResponse
{
    public string IdTaiKhoan { get; set; } = string.Empty;
    public string TenDangNhap { get; set; } = string.Empty;
    public string LoaiTaiKhoan { get; set; } = string.Empty;
    public string Ten { get; set; } = string.Empty;
    public string Cmnd { get; set; } = string.Empty;
    public string Sdt { get; set; } = string.Empty;
    public string? NgaySinh { get; set; }
    public string GioiTinh { get; set; } = string.Empty;
    public string? ChuNhiemLopId { get; set; }
    public List<string> PhanCongKeys { get; set; } = new();
}
